demo/app/assets/stylesheets/previews.scss

```diff
@@ -7,7 +7,8 @@
 }
 
 .component-preview--breadcrumbs,
-.component-preview--navigation {
+.component-preview--navigation,
+.component-preview--header {
```

Comment:
Updated these rules so the header component examples also have their margin removed. This is why the header visual regression snapshots have updated.